lib/auth/login/cubits/login_cubit.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:bloc/bloc.dart';
+import 'package:flex_storefront/auth/auth_repository.dart';
+import 'package:flex_storefront/auth/login/cubits/login_state.dart';
+import 'package:flex_storefront/shared/bloc_helper.dart';
+import 'package:get_it/get_it.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+const REMEMBER_EMAIL_KEY = 'remember_me_email';
+
+class LoginCubit extends Cubit<LoginState> {
+  LoginCubit()
+      : super(LoginState(
+          status: Status.initial,
+        )) {
+    init();
+  }
+
+  void init() async {
+    final email =
+        GetIt.instance<SharedPreferences>().getString(REMEMBER_EMAIL_KEY);
+    emit(state.copyWith(email: email));
+  }
+
+  Future<void> emailAndPasswordSignIn({
+    required String email,
+    required String password,
+  }) async {
+    try {
+      // start loading
+      emit(state.copyWith(
+        status: Status.pending,
+        email: email.trim(),
+      ));
+
+      // save data if remember me is checked
+      if (state.rememberMe) {
```

Comment:
"Remember me" means "stay connected" right? Not keeping the prefilling the email field 🙂

Reply:
"Remember Me" is prefilling form fields.
"Keep Me Signed In" is the stay connected functionality you are referring to.. which is always on by default (for apps).